share/p2p/v1/pb/share.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package share.p2p.v1;
+
+message GetSharesByNamespaceRequest{
+  bytes root_hash = 1;
+  repeated bytes row_roots = 2;
+  bytes namespace_id = 3;
+  int64 max_shares = 4;
+  bool collect_proofs = 5;
```

Comment:
I agree, do you think it would be worth to place verification login on the level of new Getter implementation, rather than client/server itself? So for handler side data coming from getter should be already verified, and for requester it will return what it got, and the caller will need to verify.

Reply:
I've added verification on client side. As for server, I think it is up to Getter interface implementation to verify returned data or not.

Reply:
I've added verification on client side. As for server, I think it is up to Getter interface implementation to verify returned data or not.